pkg/nodeagent/controller/add.go

```diff
@@ -46,5 +47,11 @@ func AddToManager(cancel context.CancelFunc, mgr manager.Manager, cfg *config.No
 		return fmt.Errorf("failed adding token controller: %w", err)
 	}
 
+	if err := (&lease.Reconciler{
```

Comment:
Please add some short documentation for this controller to https://github.com/gardener/gardener/blob/master/docs/concepts/node-agent.md#controllers (similar to https://github.com/gardener/gardener/blob/master/docs/concepts/gardenlet.md#lease-reconciler)

Reply:
Still missing